solidity/contracts/helpers/TestLiquidityProtection.sol

```diff
@@ -52,8 +52,8 @@ contract TestLiquidityProtection is LiquidityProtection, TestTime {
     }
 
     function averageRateTest(IDSToken _poolToken, IERC20Token _reserveToken) external view returns (uint256, uint256) {
-        Fraction memory rate = reserveTokenAverageRate(_poolToken, _reserveToken, true);
-        return (rate.n, rate.d);
+        (, , uint256 rateN, uint256 rateD) = reserveTokenRates(_poolToken, _reserveToken, true);
```

Comment:
I don't quite get this - it's the average rate test function that calls the non average rate, no?

Reply:
Function `reserveTokenRates` returns the spot rate (2 values) and the average rate (2 values).
Here, we ignore the first 2 values and use the last 2 values.